scenario.py

```diff
@@ -195,6 +203,14 @@ def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1, is_dry_ru
         self.scenario_id = scenario_id
         self.n_repeat = n_repeat
 
+        if "is_quick_demo" in params:
+            self.is_quick_demo = params["is_quick_demo"]
+        else:
+            self.is_quick_demo = False  # default
+
+        if self.is_quick_demo and self.proportion_dataset < 1:
+            raise Exception("Don't start a quick_demo without the full dataset")
```

Comment:
Donc on pourrait intérgrer dans le `if "is_quick_demo" in params:` et raise si `self.proportion_dataset < 1`?

Reply:
Je ne suis pas sûr d'avoir compris, j'ai modifié comme ça:
https://github.com/SubstraFoundation/distributed-learning-contributivity/blob/f3c79f1fc5064ec9fd4638fbd01ed345f68e3df3/scenario.py#L210-L215

Reply:
Is the`if self.is_quick_demo` needed line 212 ?

Reply:
We could write in the config.yml
is_quick_demo: - False